### ISSUE TITLE
wct with dynamic imports

### ISSUE THREAD

**Issue by fearphage** [Polymer/web-component-tester#133]
I'm trying to figure out how to use `wct` from the command line but add in dynamic import elements (`<link rel="import".../>`). I don't see it in the `generated.html` file. Is it possible? Am I just missing it?

**Comment by nevir** [comment]
Hmm, bit confused. Do you want to be able to `<link rel="import".../>`, but you have all of your tests as `.js` files?

**Comment by fearphage** [comment]
That is correct. I want to have the imports available so I can create elements and append them if needed.

Much like `core-ajax`, I wan to test functionality more than presentation.

**Comment by nevir** [comment]
Hmm, yeah, there's currently no way to inject additional resources into the generated index at the moment. I see two options for now:

1. You could write all your tests in `.html` files (so that you can express the import), but the duplication will probably lead to #131 

2. [Hand craft the test index](https://github.com/Polymer/web-component-tester#nested-suites) w/ all the resources you want available to your tests
